### DISCUSSION TITLE
boosts not counted globally

### DISCUSSION THREAD

**Discussion by skyerjoe** [mastodon/mastodon#23370] [Q&A]
Why is the amount of boosts from a post  related to the instance  and not globally counted over all ones.

Best regards

**Comment by trwnh** [comment]
there is no way to know every single server. no one server has a complete view of the entire network. a server can only show what it knows.

**Reply by skyerjoe** [reply]
Did an sever count boosts  based only on his instance or summed up the amount of the boosts based on  instances which the server is collaborating.

**Reply by trwnh** [reply]
@skyerjoe it's based on what reaches it via federation. if you load the status on its original server, you can see the total boosts which should be reasonably complete.
